src/domain/FlowRunner.ts

```diff
@@ -537,6 +542,7 @@ export class FlowRunner implements IFlowRunner, IFlowNavigator, IPromptBuilder {
     const originInteractionId = last(nestedFlowBlockInteractionIdStack)
     const originInteraction = originInteractionId != null ? this._contextService.findInteractionWith(originInteractionId, ctx) : null
 
+    // inflate interaction + container cursor
```

Comment:
Is this comment necessary? Is so, how could it be made into code? (as comments become stale, and indicate that clarity/naming needs to be better, or that a piece of code needs to be extracted into its own function)